lib/pengine/pe_output.c

```diff
@@ -0,0 +1,283 @@
+/*
+ * Copyright 2004-2018 Andrew Beekhof <andrew@beekhof.net>
```

Comment:
Our copyright notice changed recently, see for example lib/common/output.c

Reply:
Done.